src/test/kotlin/blackjack/view/InputViewTest.kt

```diff
@@ -0,0 +1,27 @@
+package blackjack.view
+
+import blackjack.view.InputView.Parser
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class InputViewTest {
+    @Test
+    fun `name should be parsed correctly`() {
+        val input = "mina, guri, ella"
+        val parsed = InputView.Parser.playerNames(input)
```

Comment:
```suggestion
        val parsed = Parser.playerNames(input)
```

We can remove this :)